src/framework/mpas_timekeeping.F

```diff
@@ -12,6 +12,7 @@ module mpas_timekeeping
    use mpas_dmpar
    use mpas_threading
    use mpas_log
+   use mpas_string_utils
```

Comment:
I mentioned that we're slowly improving, and here may be an example of where the existing code doesn't follow our current best practice of using the `only` clause with `use` statements. There's no need to modify the other, pre-existing `use` statements here, but for the `mpas_string_utils` module, we could update this to be:
```
   use mpas_string_utils, only : mpas_split_string
```